staff/pink/ConnectTaskApi/TaskApp-pug/src/index.js

```diff
@@ -0,0 +1,80 @@
+require('dotenv').config()
```

Comment:
Como ponéis a mano el puerto, no es necesario esta linea, aunque creemos que vale la pena incluir el archivo .env